js/activity.js

```diff
@@ -4702,8 +4702,9 @@ function Activity() {
             };
 
             this.showMusicBlocks = function() {
-                this.planet = this.getCurrentProjectName();
-                document.querySelector('title').textContent = this.planet;
+                var globalTitleS = this.getCurrentProjectName();
+                console.log(globalTitleS);
```

Comment:
Please remove the console output.

Reply:
Where else is globalTitleS used? Seems like you could directly assign a value to document.title.